tests/dir.props

```diff
@@ -114,6 +114,11 @@
     <PackageVersionArg>/p:CoreClrPackageVersion=$(VersionToRestore)</PackageVersionArg>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' != ''">$(__RuntimeId)</RuntimeIdentifiersOverwrite>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' == ''">win-$(__BuildArch)</RuntimeIdentifiersOverwrite>
```

Comment:
Any reason you cannot move the setting of TargetRID into dir.props and use exactly that property instead of creating another one?